user_interaction/static/themes/kinjin-theme.css

```diff
@@ -0,0 +1,77 @@
+@import url("https://fonts.googleapis.com/css2?family=Mochiy+Pop+One&display=swap");
+body {
+    font-family: "Mochiy Pop One", sans-serif;
+}
+/* Logo Knights verstoppen */
+.navbar-brand {
+    visibility: hidden;
+    width: auto;
+}
+/* add Kinjin Knights logo */
+.navbar-brand::before {
+    visibility: visible;
+    content: url(./images/kinjin/KinjinSpoofH70px.png);
+    width: 69px;
+}
+/* hide "Squire" */
+.brand-text {
+    visibility: hidden;
+    font-size: 0px;
+}
+/* Add Squire but now in Japanese */
+.brand-text::before {
+    content: "スクワイア";
+    visibility: visible;
+    font-size: 30px;
+}
```

Comment:
Is this still needed, since we're replacing everything with Javascript?

Reply:
Yep. In de header kon ik niets aanpassen. I tried. 

Als het echt moet kan ik met JavaScript best wel de tekst aanpassen door wat ID's aan toe te voegen. Maar dit is wat creatiever.

Reply:
fixed